homeassistant/components/google/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  config-flow:
+    status: done
+    comment: Integration uses an OAuth config flow.
+  brands: done
+  dependency-transparency: todo
+  common-modules: done
+  has-entity-name: done
+  action-setup:
+    status: todo
+    comment: |
+      Actions are current setup in `async_setup_entry` and need to be moved
+      to `async_setup`.
+  appropriate-polling: done
+  test-before-configure: done
+  entity-event-setup:
+    status: exempt
+    comment: Integration does not subscribe to events.
+  unique-config-entry: done
```

Comment:
[Here](https://github.com/home-assistant/core/blob/355e80aa56cf087f7b5b545e4209b2cb718eea87/homeassistant/components/google/__init__.py#L198)? My worry would be that the code would fail later if no unique id is set for old cases. Either way i think some code needs to handle the case and it could either raise an error for the user to reinstall or just fix it, and i think just fixing it still seems ok.